### ISSUE TITLE
[Fixed in GH-6052] safari firefox: Missing Visual Focus on Checkout: Review Order

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4327]
![Screen Shot 2019-08-29 at 7 28 30 PM](https://user-images.githubusercontent.com/15113219/63983133-62e9bf00-ca93-11e9-9cf7-8bf320ba7e60.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔ (note: 'place order' button disabled when checkbox not checked is fixed in a11y branch)
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome: OK

Missing these:
- "Edit shipping address" on firefox & safari
- Terms & Conditions on firefox & safari
- checkbox on safari
- Place Order button on safari

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 4 15 49 PM](https://user-images.githubusercontent.com/15113219/73491114-e62d7a80-437b-11ea-80ef-ebfa5fc47852.png)
![Screen Shot 2020-01-30 at 4 16 03 PM](https://user-images.githubusercontent.com/15113219/73491115-e62d7a80-437b-11ea-81fe-f9d95e83847a.png)
![Screen Shot 2020-01-30 at 4 16 28 PM](https://user-images.githubusercontent.com/15113219/73491112-e62d7a80-437b-11ea-8f0e-b0842ba53be2.png)
![Screen Shot 2020-01-30 at 4 16 14 PM](https://user-images.githubusercontent.com/15113219/73491113-e62d7a80-437b-11ea-951d-89033beb165c.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
